bundle/tests/scorecard/config.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: scorecard.operatorframework.io/v1alpha3
```

Comment:
are these tests passing? I would prefer to add scorecard in a separate PR with a proper CI execution.

Reply:
I don't know this was added by the `operator-sdk` tool. The scorecard test is out of the scope of this PR.